demo/index.js

```diff
@@ -48,7 +48,7 @@ const createOptions = (fontSize: string) => {
   };
 };
 
-class _CardForm extends React.Component {
```

Comment:
Hi! I think the right way to type this in 0.54 is to pull props (and state, if applicable) out from the _CardForm, like this:

```
type Props = {
  fontSize: string,
  stripe: StripeProps,
};

class _CardForm extends React.Component<Props> { ... }
```

Reply:
Ah! I see you've done this below for the actual lib code :).